app/backend/app.py

```diff
@@ -351,7 +473,23 @@ async def setup_clients():
         query_speller=AZURE_SEARCH_QUERY_SPELLER,
     )
 
+    current_app.config[CONFIG_CHAT_APPROACH] = ChatReadRetrieveReadApproach(
```

Comment:
Just moved this around as it was weirdly placed before. (It was Ask, then GPT4V, then Chat. Now its Chat,Ask,GPT4v)